lib/utils/util.py

```diff
@@ -688,19 +688,16 @@ def find_delimiter_in(value):
 
 
 def convert_edition_to_shortform(edition):
-    """Convert edition to shortform Enterprise or Community or N/E"""
+    """Convert edition to shortform Enterprise or Community or Federal or N/E"""
 
-    if (
-        edition.lower() in ["enterprise", "true", "ee"]
-        or "enterprise" in edition.lower()
-    ):
+    if "enterprise" in edition.lower():
```

Comment:
Shall we simply this logic further. Worst case scenario, we are doing the edition.lower() 3 times. We should just do it once and do a switch statement instead of 3 if conditions.